browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -12,6 +12,82 @@ const STORAGE_FOLDER_PLACEHOLDER = ':storage'
 const DESTINATION_FOLDER = 'attachments'
 const PATH_SEPARATORS = escapeStringRegexp(path.posix.sep) + escapeStringRegexp(path.win32.sep)
 
+function getImage (file) {
```

Comment:
There is no documentation at all.
Nor are there any tests i see..

Reply:
Ok, I will add it later